docs/builds.rst

```diff
@@ -186,6 +191,31 @@ key to the respective dictionaries of the build steps.
 Specifying the working directory for other steps works analogously to specifying the working directory for the
 configuration step (as seen above).
 
+Extra Paths
+-----------
+
+For many UNIX packages it is standard to install the executable in the ``@THIS_PREFIX_DIR@/bin`` directory.
+This is why simexpal only checks those directories by default when looking for an executable. This might
+not always be the case, especially when using a custom build system. To cover those cases, we specify the
```

Comment:
Avoid two consecutive sentences starting with `this`.

Reply:
`... However, this assumption might not always be correct, for example, when using a custom build system.`

Reply:
done